proto/frequenz/api/weather/weather.proto

```diff
@@ -88,6 +88,7 @@ message ForecastRequest {
   // The end of the period for which the forecast is being requested.
   google.protobuf.Timestamp end_ts = 4;
 
+  // Optional.
```

Comment:
Would be nice if the default behaviour were specified here.

Reply:
no application to the stream @TalweSingh